aten/src/ATen/native/native_functions.yaml

```diff
@@ -764,6 +764,10 @@
 
 - func: conv2d(Tensor input, Tensor weight, Tensor? bias=None, int[2] stride=1, int[2] padding=0, int[2] dilation=1, int groups=1) -> Tensor
 
+- func: conv2d_prepack(Tensor weight, Tensor? bias=None, int[2] stride=1, int[2] padding=0, int[2] dilation=1, int groups=1, float output_min=-HUGE_VALF, float output_max=HUGE_VALF) -> Tensor
```

Comment:
Seems the code generator cannot handle HUGE_VALF or std::numeric_limits<float>::infinity() on certain builds.  These values are bit patterns denoting infinity and have no corresponding numerical value.  We need that support for this API to work.

Alternatively, we can 1) encode that information in the function name itself - like conv2d_preack_relu and conv2d_prepack_relu6 which is not as clean in my opinion.  Or, 2) hard code these values to a very large floating-point value which should work but technically as accurate.

What is the work involved in getting HUGE_VALF or std::numeric_limits<float>::infinity() supported in the codegen?

Any preference on how to move forward?

@ezyang @gchanan @dzhulgakov @dreiss

Reply:
You can send optional arguments that, if not specified, would be converted into +/- inf in the implementation itself, similar to how e.g. clamp is implemented now https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml#L682

Reply:
Nice!! Thank you!!